infra/terraform/scripts/deploy_ec2_app.sh

```
@@ -0,0 +1,91 @@
+  if ssh "${SSH_OPTS[@]}" "${SSH_USER}@${APP_IP}" "echo ok" >/dev/null 2>&1; then
+    break
+  fi
+  sleep 5
+done
+
+ssh "${SSH_OPTS[@]}" "${SSH_USER}@${APP_IP}" "echo connected" >/dev/null
+
+scp "${SSH_OPTS[@]}" -r "${REPO_ROOT}/deploy/ec2" "${SSH_USER}@${APP_IP}:/home/${SSH_USER}/"
+scp "${SSH_OPTS[@]}" "${APP_CONFIG_SOURCE}" "${SSH_USER}@${APP_IP}:/home/${SSH_USER}/application-${SPRING_PROFILE}.yml"
+
+ssh "${SSH_OPTS[@]}" "${SSH_USER}@${APP_IP}" bash <<EOF
```

Shellcheck (0.11.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)